piqueserver/game_modes/push.py

```diff
@@ -161,24 +166,31 @@ def reset_intel_position(protocol, team):
 
 @command(admin_only=True)
 def resetintel(connection, value):
+    """
+    Manually reset the blue or green team's intel
+    /resetintel <team>
+    """
     team = get_team(connection, value)
     reset_intel_position(connection.protocol, team)
 
 
-@command('r')
-def respawn(connection):
-    if not ALLOW_RESPAWN_COMMAND.get():
-        return
-    if connection.world_object is not None and not connection.world_object.dead:
-        if (connection.last_spawn_time is None or connection.last_spawn_time +
-                RESPAWN_CMD_DELAY.get() <= get_now_in_secs()):
-            if has_flag(connection):
-                connection.drop_flag()
-            connection.spawn()
-        else:
-            connection.send_chat(
-                "Please wait %s seconds before using this command again."
-                % (connection.last_spawn_time + RESPAWN_CMD_DELAY.get() - get_now_in_secs()))
+if ALLOW_RESPAWN_COMMAND.get():
```

Comment:
why was this change made?

The old code allowed enabling/disabling the command at runtime. The new one does not.

Reply:
Oh, wasn't aware of that. The reason for that change was that the command would still appear in the list of commands or with "/help r" while being disabled. Nevermind then though, should i redo this PR without that commit?

Reply:
I think it's probably useful to a more useful "respawning is disabled" message or similar.

Or perhaps we could add some better way to disable commands. For example, using the permission system. I should re-visit #475 for that.